stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterExecutorHolder.java

```diff
@@ -16,5 +16,21 @@
  * A holder class for the executor service used for pretty printing related tasks
  */
 class AsyncPrettyPrinterExecutorHolder {
-  public static final ExecutorService sExecutorService = Executors.newCachedThreadPool();
+  private static ExecutorService sExecutorService;
+
+  public static void init() {
```

Comment:
In fact, I'm not sure we should have this stashed into a static class. Right now, `ResponseBodyFileManager` has a dependency on the fact that `DownloadingAsyncPrettyPrinterFactory` has called `init` before it calls `getExecutorService`. Why not just have the owner of `ResponseBodyFileManager` create the `ExecutorService` and pass it down?